scipy/stats/_multivariate.py

```diff
@@ -786,6 +786,39 @@ def entropy(self, mean=None, cov=1):
         dim, mean, cov_object = self._process_parameters(mean, cov)
         return 0.5 * (cov_object.rank * (_LOG_2PI + 1) + cov_object.log_pdet)
 
+    def fit(self, x):
+        """Fit the multivariate normal to data `x`.
+
+        Parameters
+        ----------
+        x : ndarray (m, n)
+            Data the distribution is fitted to. Must have two axes.
+            The first axis of length `m` represents the number of vectors
+            the distribution is fitted to. The second axis of length `n`
+            determines the dimensionality of the fitted distribution.
```

Comment:
I don't think single backticks are really appropriate for `m` and `n` because they are not "variables" in the same sense that `x` is a variable. I would prefer to make them render as math, but others wouldn't, and they wouldn't render as math in the parameter type (I don't think?), so  I think I will ignore this.